inst/include/frogger.hpp

```diff
@@ -0,0 +1,121 @@
+#pragma once
+
+#include <iostream>
+#include "types.hpp"
+
+const int MALE = 0;
```

Comment:
This is pretty nasty - you'll have a sense of how widepread this is across the code as to if we need to keep these as `#defines`

Reply:
There are about 7 constants like this that Jeff has in the full model. Should I switch that to a #define?